<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard_newBudget</class>
 <widget class="QDialog" name="dashboard_newBudget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>775</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="newBudgetText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 253, 51);
font-size: 36px;
weight: 500;
</string>
   </property>
   <property name="text">
    <string>Set a new budget</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nameText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>75</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-size: 24px;
weiht: 300;
</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>177</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-size: 24px;
weight: 300;</string>
   </property>
   <property name="text">
    <string>Set an amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="writeName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>285</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
border-radius: 20px;
padding: 0 20;
color:white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="writeAmount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>285</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 32, 32);
border-radius: 20px;
padding: 0 20;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>258</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 253, 51);
font-size: 36px;
weight: 500;</string>
   </property>
   <property name="text">
    <string>Edit budget</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentBudgetText">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>241</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-size: 24px;
weight: 300;
</string>
   </property>
   <property name="text">
    <string>Set new amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCurrentBudget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>285</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2C2B2C;
border-radius: 20px;
padding: 0 20;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="howmuchText">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-size: 24px;
weight: 300;</string>
   </property>
   <property name="text">
    <string>Set new spending</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="writeHowMuchSpent">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>285</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2C2B2C;
border-radius: 20px;
padding: 0 20;
color:white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>623</y>
     <width>322</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
border-radius:  40px;
font-size: 32px;
weight: 700;
color: rgb(182, 253, 51);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitNewBudget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 253, 51);
background-color: transparent;
weight: 600;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitEditBudget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 253, 51);
background-color: transparent;
weight: 600;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="budgetName">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>285</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2C2B2C;
border-radius: 20px;
padding: 0 20;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-size: 24px;
weiht: 300;
</string>
   </property>
   <property name="text">
    <string>Which budget</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
